manifests/nodejs.yml

```diff
@@ -816,23 +817,27 @@ tests/:
       Test_Debugger_PII_Redaction:
         '*': irrelevant
         express4: v5.39.0
+        express4-typescript: v5.39.0
```

Comment:
since the system tests CI only test on latest master branch and latest npm release, it can't check if this test passes for an older version like this one. Whenever possible, we need to run the system test locally with the old version of the library. Did you make sure of that ?

Reply:
we should do this as best practice to avoid running into parasitic failures when we run the tests on older versions (usually for debugging purposes)